### ISSUE TITLE
completed-docs doesn't cover Accessors

### ISSUE THREAD

**Issue by nixxquality** [palantir/tslint#3479]
### Bug Report

- __TSLint version__: 5.8.0
- __TypeScript version__: 2.5.3
- __Running TSLint via__: CLI / tslint-language-service

#### TypeScript code being linted

```ts
class Foo {
    private innerbar: string;

    public get bar(): string {
        return innerbar;
    }
    public set bar(value: string) {
        this.innerbar = value;
    }
}

```

with `tslint.json` configuration:

```json
{
    "rules": {
        "completed-docs": [
            true, "classes", "enums", "enum-members", "functions", "interfaces", "methods", "namespaces", "properties", "types", "variables" // everything
        ]
    }
}
```

#### Actual behavior
Only foo and innerbar get warnings about missing documentation

#### Expected behavior
bar should also get a warning.

**Comment by reduckted** [comment]
If I remember correctly, when I encountered this same problem about 12 months ago, I discovered that this is because TypeScript doesn't expose (?) those properties when walking the syntax tree. Or something like that. Maybe it's changed since then.

I recall finding an issue about this at the time (I think it was in the TypeScript repo), but I'm not having any luck finding it now. 😞

**Comment by ajafff** [comment]
Now that we no longer rely on `SyntaxWalker` do visit nodes, we can simply add a condition for `isAccessorDeclaration(node)` and check for JSDoc.
In addition there is some special handling needed, because you probably don't want to enforce JSDoc on both accessors of a property.

**Comment by reduckted** [comment]
I'm looking at this at the moment, and the change is very simple. [This `switch` statement](https://github.com/palantir/tslint/blob/cbdcfb5b7e82eacb4d3c97e06e678d1169684030/src/rules/completedDocsRule.ts#L315) just needs two extra case statements to handle `GetAccessor` and `SetAccessor` nodes.

Currently, because we are using the type checker, the documentation comments from the getter and setter are merged, so we don't need special handling for the getter having comments and the setter not (for example). That will change when we get rid of the type checker in #3376.

If we're happy to fix this while still using the type checker, then I'll go ahead and make a PR.

**Comment by ajafff** [comment]
@reduckted Go for it. Changing that when dropping the type checker should be reasonably simple.

I guess we just show an error on both accessor if none of them has a documentation comment?

**Comment by reduckted** [comment]
> I guess we just show an error on both accessor if none of them has a documentation comment?

Yep, that's exactly what it does.

**Comment by ajafff** [comment]
Fixed by #3497
